node-module-system/calculate.js

```diff
@@ -0,0 +1,23 @@
+import add from './add.js';
+
+import subtract from './subtract.js';
+
+import divide from './divide.js';
+
+import multiply from './multiply.js';
+
+if (process.argv[3] === 'plus') {
```

Comment:
You are repeating the `process.argv[3]` expression multiple times. It would be nice to extract it and avoid the repetition.